R/pMaxCombo.R

```diff
@@ -23,12 +23,12 @@ NULL
 #' MaxCombo p-value
 #'
 #' \code{pMaxCombo()} computes p-values for the MaxCombo test
-#' based on output from \code{simtrial::tenFHcorr()}.
```

Comment:
Hi @lili-ling-msd , could you please revert `wlrcorr()` back as `tenFHcorr()`?

Reply:
sure